client/src/partials/admin/creditor_groups/creditor_groups.html

```diff
@@ -0,0 +1,29 @@
+<div class="flex-header">
+  <div class="bhima-title">
+
+    <ol class="headercrumb">
+      <li class="static">{{ "TREE.FINANCE" | translate }}</li>
+      <li class="title">{{ "TREE.CREDITOR_GROUP" | translate }}
+        <span class="label label-info" ng-if="CreditorGroupCtrl.isUpdateState">{{ 'FORM.BUTTONS.EDIT' | translate }}</span>
```

Comment:
Instead of `label` try the class `badge`.  It has better line-height behavior and is a bit bigger.